<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 6.1 (Python 3.10.5 on win32)" generated="20231213 14:16:29.642" rpa="false" schemaversion="4">
<suite id="s1" name="Add To Cart Process" source="C:\Users\namra.kanwal\PycharmProjects\ebayWebsite\tests\add_to_cart_process.robot">
<test id="s1-t1" name="Search the desired product" line="9">
<kw name="Start testcase" library="CommonFunctionality">
<msg timestamp="20231213 14:16:29.892" level="TRACE">Arguments: [  ]</msg>
<kw name="Open Browser" library="SeleniumLibrary">
<arg>${url}</arg>
<arg>${browser}</arg>
<arg>{CHROMEDRIVER_PATH}= C:/Users/namra.kanwal/Downloads/chromedriver_win32 (2)/chromedriver.exe</arg>
<doc>Opens a new browser instance to the optional ``url``.</doc>
<msg timestamp="20231213 14:16:29.892" level="TRACE">Arguments: [ 'https://www.saucedemo.com/inventory.html' | 'chrome' | '{CHROMEDRIVER_PATH}= C:/Users/namra.kanwal/Downloads/chromedriver_win32 (2)/chromedriver.exe' ]</msg>
<msg timestamp="20231213 14:16:29.892" level="INFO">Opening browser 'chrome' to base url 'https://www.saucedemo.com/inventory.html'.</msg>
<msg timestamp="20231213 14:16:29.908" level="INFO">Executing: C:\Python\lib\site-packages\selenium\webdriver\common\windows\selenium-manager.exe --browser chrome --output json</msg>
<msg timestamp="20231213 14:16:32.324" level="WARN">Unable to obtain driver using Selenium Manager: Selenium Manager failed for: C:\Python\lib\site-packages\selenium\webdriver\common\windows\selenium-manager.exe --browser chrome --output json.
The chromedriver version cannot be discovered</msg>
<msg timestamp="20231213 14:16:32.324" level="INFO">Cannot capture screenshot because no browser is open.</msg>
<msg timestamp="20231213 14:16:32.324" level="FAIL">SeleniumManagerException: Message: Selenium Manager failed for: C:\Python\lib\site-packages\selenium\webdriver\common\windows\selenium-manager.exe --browser chrome --output json.
The chromedriver version cannot be discovered
</msg>
<msg timestamp="20231213 14:16:32.324" level="DEBUG">Traceback (most recent call last):
  File "C:\Python\lib\site-packages\SeleniumLibrary\__init__.py", line 517, in run_keyword
    return DynamicCore.run_keyword(self, name, args, kwargs)
  File "C:\Python\lib\site-packages\robotlibcore.py", line 94, in run_keyword
    return self.keywords[name](*args, **(kwargs or {}))
  File "C:\Python\lib\site-packages\SeleniumLibrary\keywords\browsermanagement.py", line 291, in open_browser
    return self._make_new_browser(
  File "C:\Python\lib\site-packages\SeleniumLibrary\keywords\browsermanagement.py", line 322, in _make_new_browser
    driver = self._make_driver(
  File "C:\Python\lib\site-packages\SeleniumLibrary\keywords\browsermanagement.py", line 777, in _make_driver
    driver = self._webdriver_creator.create_driver(
  File "C:\Python\lib\site-packages\SeleniumLibrary\keywords\webdrivertools\webdrivertools.py", line 91, in create_driver
    return creation_method(
  File "C:\Python\lib\site-packages\SeleniumLibrary\keywords\webdrivertools\webdrivertools.py", line 148, in create_chrome
    return webdriver.Chrome(
  File "C:\Python\lib\site-packages\selenium\webdriver\chrome\webdriver.py", line 82, in __init__
    service.path = DriverFinder.get_path(service, options)
  File "C:\Python\lib\site-packages\selenium\webdriver\common\driver_finder.py", line 43, in get_path
    raise err
  File "C:\Python\lib\site-packages\selenium\webdriver\common\driver_finder.py", line 40, in get_path
    path = shutil.which(service.path) or SeleniumManager().driver_location(options)
  File "C:\Python\lib\site-packages\selenium\webdriver\common\selenium_manager.py", line 101, in driver_location
    result = self.run(args)
  File "C:\Python\lib\site-packages\selenium\webdriver\common\selenium_manager.py", line 122, in run
    raise SeleniumManagerException(f"Selenium Manager failed for: {command}.\n{result}{stderr}")
selenium.common.exceptions.SeleniumManagerException: Message: Selenium Manager failed for: C:\Python\lib\site-packages\selenium\webdriver\common\windows\selenium-manager.exe --browser chrome --output json.
The chromedriver version cannot be discovered</msg>
<status status="FAIL" starttime="20231213 14:16:29.892" endtime="20231213 14:16:32.324"/>
</kw>
<kw name="Maximize Browser Window" library="SeleniumLibrary">
<doc>Maximizes current browser window.</doc>
<status status="NOT RUN" starttime="20231213 14:16:32.324" endtime="20231213 14:16:32.324"/>
</kw>
<status status="FAIL" starttime="20231213 14:16:29.892" endtime="20231213 14:16:32.324"/>
</kw>
<kw name="Enter email" library="swag_UserDefinedKeyword">
<status status="NOT RUN" starttime="20231213 14:16:32.324" endtime="20231213 14:16:32.324"/>
</kw>
<kw name="Enter Password" library="swag_UserDefinedKeyword">
<status status="NOT RUN" starttime="20231213 14:16:32.324" endtime="20231213 14:16:32.324"/>
</kw>
<kw name="Click SignIn" library="swag_UserDefinedKeyword">
<status status="NOT RUN" starttime="20231213 14:16:32.324" endtime="20231213 14:16:32.324"/>
</kw>
<kw name="Execute Javascript" library="SeleniumLibrary">
<arg>window.scrollTo(0,400)</arg>
<doc>Executes the given JavaScript code with possible arguments.</doc>
<status status="NOT RUN" starttime="20231213 14:16:32.324" endtime="20231213 14:16:32.324"/>
</kw>
<kw name="Wait Until Element Is Not Visible" library="SeleniumLibrary">
<arg>swag_UserDefinedKeyword.Verify filter results</arg>
<doc>Waits until the element ``locator`` is not visible.</doc>
<status status="NOT RUN" starttime="20231213 14:16:32.324" endtime="20231213 14:16:32.324"/>
</kw>
<kw name="Finish testcase" library="CommonFunctionality">
<status status="NOT RUN" starttime="20231213 14:16:32.324" endtime="20231213 14:16:32.324"/>
</kw>
<doc>This case search the desired product</doc>
<tag>Functional</tag>
<status status="FAIL" starttime="20231213 14:16:29.892" endtime="20231213 14:16:32.337">SeleniumManagerException: Message: Selenium Manager failed for: C:\Python\lib\site-packages\selenium\webdriver\common\windows\selenium-manager.exe --browser chrome --output json.
The chromedriver version cannot be discovered
</status>
</test>
<test id="s1-t2" name="Add to cart process" line="21">
<kw name="Start testcase" library="CommonFunctionality">
<msg timestamp="20231213 14:16:32.339" level="TRACE">Arguments: [  ]</msg>
<kw name="Open Browser" library="SeleniumLibrary">
<arg>${url}</arg>
<arg>${browser}</arg>
<arg>{CHROMEDRIVER_PATH}= C:/Users/namra.kanwal/Downloads/chromedriver_win32 (2)/chromedriver.exe</arg>
<doc>Opens a new browser instance to the optional ``url``.</doc>
<msg timestamp="20231213 14:16:32.340" level="TRACE">Arguments: [ 'https://www.saucedemo.com/inventory.html' | 'chrome' | '{CHROMEDRIVER_PATH}= C:/Users/namra.kanwal/Downloads/chromedriver_win32 (2)/chromedriver.exe' ]</msg>
<msg timestamp="20231213 14:16:32.340" level="INFO">Opening browser 'chrome' to base url 'https://www.saucedemo.com/inventory.html'.</msg>
<msg timestamp="20231213 14:16:32.342" level="INFO">Executing: C:\Python\lib\site-packages\selenium\webdriver\common\windows\selenium-manager.exe --browser chrome --output json</msg>
<msg timestamp="20231213 14:16:34.653" level="WARN">Unable to obtain driver using Selenium Manager: Selenium Manager failed for: C:\Python\lib\site-packages\selenium\webdriver\common\windows\selenium-manager.exe --browser chrome --output json.
The chromedriver version cannot be discovered</msg>
<msg timestamp="20231213 14:16:34.653" level="INFO">Cannot capture screenshot because no browser is open.</msg>
<msg timestamp="20231213 14:16:34.653" level="FAIL">SeleniumManagerException: Message: Selenium Manager failed for: C:\Python\lib\site-packages\selenium\webdriver\common\windows\selenium-manager.exe --browser chrome --output json.
The chromedriver version cannot be discovered
</msg>
<msg timestamp="20231213 14:16:34.653" level="DEBUG">Traceback (most recent call last):
  File "C:\Python\lib\site-packages\SeleniumLibrary\__init__.py", line 517, in run_keyword
    return DynamicCore.run_keyword(self, name, args, kwargs)
  File "C:\Python\lib\site-packages\robotlibcore.py", line 94, in run_keyword
    return self.keywords[name](*args, **(kwargs or {}))
  File "C:\Python\lib\site-packages\SeleniumLibrary\keywords\browsermanagement.py", line 291, in open_browser
    return self._make_new_browser(
  File "C:\Python\lib\site-packages\SeleniumLibrary\keywords\browsermanagement.py", line 322, in _make_new_browser
    driver = self._make_driver(
  File "C:\Python\lib\site-packages\SeleniumLibrary\keywords\browsermanagement.py", line 777, in _make_driver
    driver = self._webdriver_creator.create_driver(
  File "C:\Python\lib\site-packages\SeleniumLibrary\keywords\webdrivertools\webdrivertools.py", line 91, in create_driver
    return creation_method(
  File "C:\Python\lib\site-packages\SeleniumLibrary\keywords\webdrivertools\webdrivertools.py", line 148, in create_chrome
    return webdriver.Chrome(
  File "C:\Python\lib\site-packages\selenium\webdriver\chrome\webdriver.py", line 82, in __init__
    service.path = DriverFinder.get_path(service, options)
  File "C:\Python\lib\site-packages\selenium\webdriver\common\driver_finder.py", line 43, in get_path
    raise err
  File "C:\Python\lib\site-packages\selenium\webdriver\common\driver_finder.py", line 40, in get_path
    path = shutil.which(service.path) or SeleniumManager().driver_location(options)
  File "C:\Python\lib\site-packages\selenium\webdriver\common\selenium_manager.py", line 101, in driver_location
    result = self.run(args)
  File "C:\Python\lib\site-packages\selenium\webdriver\common\selenium_manager.py", line 122, in run
    raise SeleniumManagerException(f"Selenium Manager failed for: {command}.\n{result}{stderr}")
selenium.common.exceptions.SeleniumManagerException: Message: Selenium Manager failed for: C:\Python\lib\site-packages\selenium\webdriver\common\windows\selenium-manager.exe --browser chrome --output json.
The chromedriver version cannot be discovered</msg>
<status status="FAIL" starttime="20231213 14:16:32.339" endtime="20231213 14:16:34.653"/>
</kw>
<kw name="Maximize Browser Window" library="SeleniumLibrary">
<doc>Maximizes current browser window.</doc>
<status status="NOT RUN" starttime="20231213 14:16:34.653" endtime="20231213 14:16:34.653"/>
</kw>
<status status="FAIL" starttime="20231213 14:16:32.338" endtime="20231213 14:16:34.653"/>
</kw>
<kw name="Enter email" library="swag_UserDefinedKeyword">
<status status="NOT RUN" starttime="20231213 14:16:34.653" endtime="20231213 14:16:34.653"/>
</kw>
<kw name="Enter Password" library="swag_UserDefinedKeyword">
<status status="NOT RUN" starttime="20231213 14:16:34.653" endtime="20231213 14:16:34.653"/>
</kw>
<kw name="Click SignIn" library="swag_UserDefinedKeyword">
<status status="NOT RUN" starttime="20231213 14:16:34.653" endtime="20231213 14:16:34.653"/>
</kw>
<kw name="Click on desired item" library="swag_UserDefinedKeyword">
<status status="NOT RUN" starttime="20231213 14:16:34.653" endtime="20231213 14:16:34.653"/>
</kw>
<kw name="Sleep" library="BuiltIn">
<arg>2s</arg>
<doc>Pauses the test executed for the given time.</doc>
<status status="NOT RUN" starttime="20231213 14:16:34.653" endtime="20231213 14:16:34.653"/>
</kw>
<kw name="Add to cart a product" library="swag_UserDefinedKeyword">
<status status="NOT RUN" starttime="20231213 14:16:34.653" endtime="20231213 14:16:34.653"/>
</kw>
<kw name="Sleep" library="BuiltIn">
<arg>5s</arg>
<doc>Pauses the test executed for the given time.</doc>
<status status="NOT RUN" starttime="20231213 14:16:34.653" endtime="20231213 14:16:34.653"/>
</kw>
<kw name="Thankyou Message" library="swag_UserDefinedKeyword">
<status status="NOT RUN" starttime="20231213 14:16:34.653" endtime="20231213 14:16:34.653"/>
</kw>
<kw name="Finish testcase" library="CommonFunctionality">
<status status="NOT RUN" starttime="20231213 14:16:34.653" endtime="20231213 14:16:34.653"/>
</kw>
<doc>This case add a product to cart</doc>
<tag>Functional</tag>
<status status="FAIL" starttime="20231213 14:16:32.337" endtime="20231213 14:16:34.653">SeleniumManagerException: Message: Selenium Manager failed for: C:\Python\lib\site-packages\selenium\webdriver\common\windows\selenium-manager.exe --browser chrome --output json.
The chromedriver version cannot be discovered
</status>
</test>
<status status="FAIL" starttime="20231213 14:16:29.642" endtime="20231213 14:16:34.653"/>
</suite>
<statistics>
<total>
<stat pass="0" fail="2" skip="0">All Tests</stat>
</total>
<tag>
<stat pass="0" fail="2" skip="0">Functional</stat>
</tag>
<suite>
<stat pass="0" fail="2" skip="0" id="s1" name="Add To Cart Process">Add To Cart Process</stat>
</suite>
</statistics>
<errors>
<msg timestamp="20231213 14:16:29.892" level="ERROR">Error in file 'C:\Users\namra.kanwal\PycharmProjects\ebayWebsite\tests\add_to_cart_process.robot' on line 5: Error in file 'C:\Users\namra.kanwal\PycharmProjects\ebayWebsite\tests\login.robot' on line 7: Resource file with 'Test Cases' section is invalid.</msg>
<msg timestamp="20231213 14:16:32.324" level="WARN">Unable to obtain driver using Selenium Manager: Selenium Manager failed for: C:\Python\lib\site-packages\selenium\webdriver\common\windows\selenium-manager.exe --browser chrome --output json.
The chromedriver version cannot be discovered</msg>
<msg timestamp="20231213 14:16:34.653" level="WARN">Unable to obtain driver using Selenium Manager: Selenium Manager failed for: C:\Python\lib\site-packages\selenium\webdriver\common\windows\selenium-manager.exe --browser chrome --output json.
The chromedriver version cannot be discovered</msg>
</errors>
</robot>
